### ISSUE TITLE
Flesh//Blood restarted the game in a buggy state

### ISSUE THREAD

**Issue by mdog1** [PennyDreadfulMTG/modo-bugs#542]
**What is the expected behaviour of the card?**
 - Flesh // Blood was cast fused by my opponent. It was supposed to make a creature bigger with flesh, and deal damage to me with blood. I never saw the card on the stack, only the game stating that my opponent was casting Blood. My opponent claimed "I tried to fuse both targeting your face".

**What is the actual behaviour of the card?**
 - The game quickly went back to the beginning, replayed everything we did, then restarted the game in a buggy state. For example, my opponent could not see or block some of my creatures and the game stated that Sprouting Thrinax blocked Jugan, the Rising Star, when Jugan was not in the game and Gathan Raiders were unblocked. This is what the screenshot shows, since the video of what happened before the game restarted does no appear to exist. (Note: my opponent's name is Error 2105, that was not an error created by the game.)

**Screenshot**
![image](https://user-images.githubusercontent.com/40871277/42346732-96f77872-8071-11e8-904a-913447bd8b9d.png)

**Please provide a list of affected cards**
Flesh//Blood
**Card Images**
 

Affects:

**Comment by silasary** [comment]
Closing in favor of #543
